doc/modules/lda_qda.rst

```diff
@@ -107,22 +96,60 @@ In the case of QDA, there are no assumptions on the covariance matrices
 	  and the resulting classifier is equivalent to the Gaussian Naive Bayes
 	  classifier :class:`naive_bayes.GaussianNB`.
 
-Mathematical formulation of LDA dimensionality reduction
-========================================================
+LDA
+---
+
+LDA is a special case of QDA, where the Gaussians for each class are assumed
+to share the same covariance matrix: :math:`\Sigma_k = \Sigma` for all
+:math:`k`. This reduces the log posterior to:
 
-To understand the use of LDA in dimensionality reduction, it is useful to start
-with a geometric reformulation of the LDA classification rule explained above.
-We write :math:`K` for the total number of target classes. Since in LDA we
-assume that all classes have the same estimated covariance :math:`\Sigma`, we
-can rescale the data so that this covariance is the identity:
+.. math:: \log P(y=k | x) = -\frac{1}{2} (x-\mu_k)^t \Sigma^{-1} (x-\mu_k) + \log P(y = k) + Cst.
+
+The term :math:`(x-\mu_k)^t \Sigma^{-1} (x-\mu_k)` corresponds to the
+`Mahalanobis Distance <https://en.wikipedia.org/wiki/Mahalanobis_distance>`_
+between the sample :math:`x` and the mean :math:`\mu_k`. The Mahalanobis
+distance tells how close :math:`x` is from :math:`\mu_k`, while also
+accounting for the variance of each feature. We can thus interpret LDA as
+assigning :math:`x` to the class whose mean is the closest in terms of
+Mahalanobis distance, while also accounting for the class prior
+probabilities.
+
+The log-posterior of LDA can also be written [3]_ as:
+
+.. math::
 
-.. math:: X^* = D^{-1/2}U^t X\text{ with }\Sigma = UDU^t
+    \log P(y=k | x) = \omega_k^t x + \omega_{k0} + Cst.
 
-Then one can show that to classify a data point after scaling is equivalent to
-finding the estimated class mean :math:`\mu^*_k` which is closest to the data
-point in the Euclidean distance. But this can be done just as well after
-projecting on the :math:`K-1` affine subspace :math:`H_K` generated by all the
-:math:`\mu^*_k` for all classes. This shows that, implicit in the LDA
+where :math:`\omega_k = \Sigma^{-1} \mu_k` and :math:`\omega_{k0} =
+-\frac{1}{2} \mu_k^t\Sigma^{-1}\mu_k + \log P (y = k)`. These quantities
+correspond to the `coef_` and `intercept_` attributes, respectively.
+
+From the above formula, it is clear that LDA has a linear decision surface.
+In the case of QDA, there are no assumptions on the covariance matrices
+:math:`\Sigma_k` of the Gaussians, leading to quadratic decision surfaces.
+See [1]_ for more details.
+
+Mathematical formulation of LDA dimensionality reduction
+========================================================
+
+First note that the K means :math:`\mu_k` are vectors in
+:math:`\mathcal{R}^d`, and they lie in an affine subspace :math:`H` of
+dimension at least :math:`K - 1` (2 points lie on a line, 3 points lie on a
+plane, etc).
+
+As mentionned above, we can interpret LDA as assigning :math:`x` to the class
```

Comment:
```suggestion
As mentionned above, we can interpret QDA as assigning :math:`x` to the class
```

Reply:
I think this is about LDA and not QDA since we cannot perform projections with QDA?

Reply:
My mistake